src/main/resources/templates/notifications/upload-postal-document.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'actions'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Provide Documents or Evidence</title>
+</head>
+<body>
+
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/notifications/{notification_id}/provide-documents-or-evidence(notification_id=${notificationId})}"
+       class="govuk-back-link">Return to Provide Documents or Evidence</a>
+  </div>
+  <div id="main-content">
+    <h1 class="govuk-heading-l">Provide Postal Documents or Evidence</h1>
+    <p class="govuk-body-l">
+      Please provide details of the document you will be posting.
+    </p>
+    <p class="govuk-body-l">
+      * indicates required field
```

Comment:
pages should not include required asterisks anymore as part of gds design

If you ask for optional information:

in most contexts, add ‘(optional)’ to the labels of optional fields
for [radios](https://design-system.service.gov.uk/components/radios/) and [checkboxes](https://design-system.service.gov.uk/components/checkboxes/), add ‘(optional)’ to the legend

Never mark mandatory fields with asterisks.

Reply:
It looks like I accidentally pushed this in the last commit, it was just a draft for the upload screens I'm working on now. I've removed it.

I was wondering about the asterisks though, as I hadn't seen them on any other pages. Thanks for clearing that up.